core/src/main/resources/hudson/model/UsageStatistics/help-usageStatisticsCollected.jelly

```diff
@@ -48,7 +48,7 @@
                 <j:if test="${not collector.end.isBefore(now)}">
                     <dt>${collector.displayName}</dt>
                     <dd>
-                        <st:include from="${collector}" optional="true" page="description.jelly"/>
+                        <st:include it="${collector}" optional="true" page="description.jelly"/>
```

Comment:
Required to be able to access `it` (the current `Telemetry` implementation) in `description.jelly`.